### ISSUE TITLE
Parameters for highly charged molecules

### ISSUE THREAD

**Issue by matteoaldeghi** [leelasd/ligpargen#10]
- [ ] I believe this to be a bug with LigParGen 
- [x] This is a feature request

I wonder whether it is possible to allow molecules with net charge larger than +/-2 to be parameterised by the LigParGen Server?

**Comment by leelasd** [comment]
@matteoaldeghi Problem with AM1 semi-empirical model and CM1A charge model is that they  not parameterized for charges other than +2,-2,+1,-2,0. So, I suspect how well they will perform for highly charged systems.

**Comment by matteoaldeghi** [comment]
@leelasd I see the problem - allowing only certain charges, however, really limits the usability of the force field in particular when it comes to simulating things like cofactors (e.g. ATP, GTP) that are quite common. Maybe an option could be to allow charges above 2 but at the same time give a big warning about the fact the charge model was not parameterised for such systems?

I also noticed that you now provide a way to install and run ligpargen locally via command line, which is very useful. Is the charge limitation still present also when using the program via command line? My practical issue is that I am testing/comparing a few force fields for free energy calculations using Gromacs, and I would like to include OPLS too, but many of the molecules I am looking at are highly charged...

**Comment by leelasd** [comment]
@matteoaldeghi I agree with you, It would be useful to allow higher charge values but it's important to realize that molecules like ATP, GTP and other nucleotides need to be parameterized explicitly. In fact, one of my coworkers published papers on nucleotide parameters. We don't really want people to use charges that are not accurate from the web server, that makes the force field look bad. I can enable the option of higher charges for local version LigParGen (assuming they are aware of the limitations)

Regarding Local version of LigParGen, can you email Prof. Jorgensen and cc me, requesting BOSS software. Then I can send you everything you need for setting up a local version of LigParGen. Also, It would be an unfair comparison if you are comparing FF for these highly charged systems, most of them may not have been included in parameterization set 😃 My guess is most existing FF parameters are poor for these systems.
